lib/extensions.js

```diff
@@ -260,6 +261,80 @@ function getBinaryPath() {
 }
 
 /**
+ * An array of paths suitable for use as a local disk cache of the binding.
+ *
+ * @return {[]String} an array of paths
+ * @api public
+ */
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_sass_binary_cache,
+    process.env.npm_config_cache,
+    getBinaryPath(),
```

Comment:
I think this should be removed since it's not really a cache path, but the final fallback